lib/kissable/ab.rb

```diff
@@ -19,8 +19,10 @@ def initialize(test_name, groups=nil, ratios=nil)
       validate_ratios
     end
 
-    def group(cookies)
-      @cookies = cookies
+    # Assigns a test group based on the 'abid' cookie or login (email).
+    # Takes either a cookies object (Hash-like) or login (String).
+    def group(object)
+      (object.is_a? String) ? (@login = object) : (@cookies = object)
```

Comment:
Here's a related topic. For an AB instance `@seed` is only calculated once ever. The consequence is that `group` always returns the first group assigned, even if called with different cookies or logins. If we want that behavior, then I feel like we might as well cache the `@group` the same way.